src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken(props.token);
+  }, [props.token]);
+
+  const bannedAlert = () => {
+    if (!token.banned) {
+      return null;
+    }
+
+    const bannedMessage = ''; // TODO: this message is being disscussed here: https://github.com/HathorNetwork/hathor-explorer/issues/92
```

Comment:
This message is duplicated in `src/components/token/TokenMarkers.js`. We should move it somewhere else to keep it in only one place.

Reply:
It will turn into something like `t('REGULAR_BAN')`.

Reply:
The message is still duplicated, isn't it? You have this variable in both components.

Reply:
Yes, they are the same translated message, the key is the same, they will come from the same i18n entry. If the message change, we won't have to change in both files. 

Of course, i can just remove this variable and put `t('REGULAR_BAN')` directly on the html but i've decided to let it explicit that it will changed. Also, i don't think it would have any advantages to move this to a different file now just to be alone and empty there.

Reply:
done!